include/cantera/thermo/ThermoPhase.h

```diff
@@ -975,6 +974,118 @@ class ThermoPhase : public Phase
      */
     virtual void setState_SV(doublereal s, doublereal v, doublereal tol = 1.e-4);
 
+    //! Set the specific entropy (J/kg/K) and temperature (K).
+    /*!
+     * This function fixes the internal state of the phase so that the specific
+     * entropy and temperature have the value of the input parameters.
+     * It must be implemented in a derived class.
```

Comment:
I guess I'm looking for a phrasing that distinguishes between methods like ThermoPhase::pressure, which if left unimplemented causes a large number of other methods to not work as well, and this, which just means that this one method has no implementation in the base class.